spec/models/merchant_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
```

Comment:
Let's either add tests here, or remove the file. Should we be testing our models? I am guessing the answer is yes...